Library/DataSource/SimpleDataSource.swift

```diff
@@ -4,33 +4,33 @@ import UIKit
 // and uses `SimpleViewModel` for its data.
 public final class SimpleDataSource <
   Cell: ValueCell,
-  Value: Any
+  Value: Any>: ValueCellDataSource
   where
-  Cell.Value == Value>: ValueCellDataSource {
+  Cell.Value == Value {
```

Comment:
these few lines is just cause `where` clauses for generic constraints have been moved outside the `<>` in swift 3